README.md

```diff
@@ -23,8 +23,13 @@ Please check out common DSC Resources [contributing guidelines](https://github.c
 * **xSQLServerMaxDop** resource to manage MaxDegree of Parallism for SQL Server
 * **xSQLServerMemory** resource to manage Memory for SQL Server
 * **xSQLServerPowerPlan** resource to manage windows powerplan on SQL Server
+<<<<<<< HEAD
```

Comment:
merge error +<<<<<<< HEAD should not be in the file   look for >>>>>> lines as well